runtimes/eden/src/pallets_system.rs

```diff
@@ -96,11 +97,16 @@ impl pallet_balances::Config for Runtime {
 	type MaxReserves = ();
 	type ReserveIdentifier = [u8; 8];
 	type Balance = Balance;
-	type DustRemoval = CompanyReserve;
+	type DustRemoval = (); // CompanyReserve; //type NegativeImbalance = <Balances as Currency<AccountId>>::NegativeImbalance;
```

Comment:
- [ ] This had a value before

Reply:
Would the dust still go the company reserve?

Reply:
No dust is lost here:

![image](https://github.com/NodleCode/chain/assets/3679593/cf3f9d1c-1e04-42ac-bc2d-bad9ff8f073b)

Checked with an ED of 100

Reply:
Confirmed we change to burning dust.